src/System.Device.Gpio.Tests/Mcp3008Tests.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+using System.Device;
+using System.Device.Pwm;
+using Iot.Device.Adc;
+
+using Xunit;
+
+using static System.Device.Gpio.Tests.SetupHelpers;
+
+namespace System.Device.Gpio.Tests
+{
+    public class Mcp3008Tests
```

Comment:
do we also plan to write tests that actually test the device functionality? I ask because technically we are not really testing in here if SPI is working or not.

Reply:
FunctionalTests do test MCP3008